portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -733,6 +733,17 @@ definition {
 			value1 = "${fieldValidationErrorMessage}");
 	}
 
+	macro viewFieldRepeatableButton {
+		key_fieldName = "${fieldName}";
+
+		for (var number : list "${key_repeatableFieldNumber}") {
```

Comment:
@leticiachiu how about renaming it to `key_repeatableFieldsNumber`? (`fields` instead of just `field`)